dll/shellext/stobject/power.cpp

```diff
@@ -107,7 +110,25 @@ static HICON DynamicLoadIcon(HINSTANCE hinst)
     {
         index = Quantize(PowerStatus.BatteryLifePercent);
         hBatIcon = LoadIcon(hinst, MAKEINTRESOURCE(br_icons[index]));
-        g_strTooltip.Format(IDS_PWR_PERCENT_REMAINING, PowerStatus.BatteryLifePercent);
+
+        if (PowerStatus.BatteryLifeTime != BATTERY_UNKNOWN_TIME)
+        {
+            uiHour = PowerStatus.BatteryLifeTime / HOUR_IN_SECS;
+            uiMin = (PowerStatus.BatteryLifeTime & HOUR_IN_SECS) / MIN_IN_SECS;
```

Comment:
Does `&` operator work correctly here? It's operating on bits, not on numbers, you probably want `%` :
```suggestion
            uiMin = (PowerStatus.BatteryLifeTime % HOUR_IN_SECS) / MIN_IN_SECS;
```

Reply:
SHIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIITTTTTTTTTTT A TYPO.... Thanks for noticing that...!